Nodejs/Product/Npm/PackageJsonFactory.cs

```diff
@@ -67,5 +67,10 @@ public static IPackageJson Create(string fullPathToFile)
 
             return null;
         }
+
+        public static bool IsPackageJsonFile(string filePath)
+        {
+            return filePath?.EndsWith(NodejsConstants.PackageJsonFile, StringComparison.OrdinalIgnoreCase) == true;
```

Comment:
>EndsWith [](start = 29, length = 8)

Would it be more accurate to use `Path.GetFileName`?

Reply:
I figured I would avoid creating a new string, but as long as I have no evidence that's actually a problem I won't